sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -353,8 +348,10 @@ def fetch_availability_zone_mappings() -> pd.DataFrame:
         use1-az2          us-east-1a
     """
     regions = list(get_enabled_regions())
-    az_mappings = [_get_availability_zones.remote(r) for r in regions]
-    az_mappings = ray.get(az_mappings)
+    # Use ThreadPool instead of Pool because this function can be called within
+    # a multiprocessing.Pool, and Pool cannot be nested.
+    with multiprocessing.pool.ThreadPool() as pool:
```

Comment:
This doesn't look nested?

Reply:
In #1691, we will use multiprocessing in `cli.status` which may further call this function within the multiprocessing Pool.